src/css/components/card.scss

```diff
@@ -0,0 +1,27 @@
+@import '../../../node_modules/uswds/src/stylesheets/core/defaults.scss';
+@import '../override_vars.scss';
+@import '../core.scss';
+
+.card-home-contents {
```

Comment:
I wan this to be more generic then for "home". If it's not being used anywhere else, why can't we just call it `card`?